lib/utils.js

```diff
@@ -48,10 +50,40 @@ ${args.length} arguments.`;
     }
 }
 
+function getTranslations(filename, translationContext) {
```

Comment:
I've designed this to allow multiple .po files even though the settings dont yet support it, as I anticipate that we would want to allow checking multiple files, either for separate .po files for different languages, or more likely different .pot files for different domains.